src/FeatureDetection.js

```diff
@@ -20,4 +22,8 @@ function speechRecognitionApiIsAvailable(): boolean {
     return !!((window: WindowSubset).webkitSpeechRecognition);
 }
 
-export { bluetoothApiIsAvailable, speechRecognitionApiIsAvailable };
+function webAudioApiIsAvailable(): boolean {
+    return !!(window.AudioContext || window.webkitAudioContext);
```

Comment:
You add `AudioContext` and `webkitAudioContext` to the `WindowSubset` type but then don't use it in the check. 
And I think in this case, the "!!" isn't needed to coerce to a boolean, as we are using the "||" operator.

Reply:
I think it is, or just returns the first truthy one.  From the console:

```
(window.AudioContext || window.webkitAudioContext )
ƒ AudioContext() { [native code] }
```

With the double bangs, the output is correct:

```
!!(window.AudioContext || window.webkitAudioContext )
true
```

Reply:
Ah yes. Thanks Tony,